config/core/configmaps/autoscaler.yaml

```diff
@@ -187,6 +187,11 @@ data:
     # or the "autoscaling.knative.dev/initialScale" annotation, can be set to 0.
     allow-zero-initial-scale: "false"
 
+    # min-scale is the cluster-wide default value for the min scale of a revision,
+    # unless overridden by the "autoscaling.knative.dev/minScale" annotation.
+    # This value must be greater than 0 if enable-scale-to-zero is false.
```

Comment:
I think(?) this might not work well with the case where someone upgrades from a previously valid config with scale-to-zero false and no value for this key.

My instinct is to keep this independent of scale to zero: if scale-to-zero is false and this is 0, we just won't scale to zero. It [doesn't look like](https://github.com/knative/serving/blob/dd83fd97de80788d81e69f45b69bc30d7f4b1866/pkg/apis/autoscaling/annotation_validation.go#L177) (on a quick scan anyway) we validate the corresponding annotation is >0 when scale to zero is false